ansible/home/ec2-user/.env.j2

```diff
@@ -0,0 +1,16 @@
+  GNU nano 5.8                                                                                             .env.j2                                                                                                        
```

Comment:
I think this line has probably snuck in somewhere - shouldn't cause any harm but a good habit to cut anything extraneous